vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -72,6 +72,8 @@ $vsWorkloads = @('Microsoft.VisualStudio.Workload.ManagedDesktop',
     'Microsoft.VisualStudio.Workload.NativeDesktop',
     'Microsoft.VisualStudio.Workload.Universal');
 
+$vsver = "16.5"
```

Comment:
nit: commen that this is the minimum VS version string

Reply:
```suggestion
# The minimum VS version to check for
$vsver = "16.5"
```